docs/ref-manual/cvl/invariants.md

```diff
@@ -29,6 +29,7 @@ invariant ::= "invariant" id
 
 preserved_block ::= "preserved"
                     [ method_signature ]
+                    [ "fallback()" ]
```

Comment:
This should probably be "fallback" "(" ")".  Also, fallback should be part of `method_signature`; with the way it's written here you can write something like "preserved f(uint x) fallback () { ... }".  Better to add
```
method_signature ::= id "(" types ")"
     | "fallback" "(" ")"
```

Reply:
it should probably actually be
```
  id "(" [ evm_type [ id ] { "," evm_type [ id ] } ] ")"
| "fallback" "(" ")"
```

for the method_signature definition

Reply:
done